db/migrate/20220217100338_create_posts.rb

```diff
@@ -3,7 +3,8 @@ def change
     create_table :posts do |t|
       t.string :title
       t.text :text
-
+      t.integer :comments_counter, :default => 0
+      t.integer :likes_counter, :default => 0
       t.timestamps
       t.integer :comments_counter, :default => 0
       t.integer :likes_counter, :default => 0
```

Comment:
- [ ] I got an error below while trying to run your your migrations. Kindly remove the dublicate `t.integer :comments_counter, :default => 0` and `t.integer :likes_counter, :default => 0` to fix this error.

<img width="442" alt="Code_uwtBbEITbO" src="https://user-images.githubusercontent.com/81468675/155996965-e897ecb6-0c81-421a-bdf7-248af8269bb0.png">